backend/api.py

```
@@ -316,173 +314,13 @@ async def login_for_access_token(form_data: UserLogin):
-                websocket, "Disconnected during auth"
-            )
-    except Exception as e:
-        logger.exception("Unhandled WebSocket error")
-        if authenticated and user_id:
-            await memory_timeline.unsubscribe(websocket)
-            await connection_manager.disconnect(websocket, user_id)
-        else:
-            await connection_manager.reject_connection(websocket, f"Error: {str(e)}")
+async def websocket_timeline(
+    websocket: WebSocket,
+    service=Depends(get_websocket_service),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)